src/components/Search/index.tsx

```diff
@@ -446,43 +453,45 @@ function Search({queryJSON, currentSearchResults, lastNonEmptySearchResults, onS
     const shouldShowSorting = !Array.isArray(status) && !shouldGroupByReports;
 
     return (
-        <SearchList
-            ref={handleSelectionListScroll(sortedSelectedData)}
-            data={sortedSelectedData}
-            ListItem={ListItem}
-            onSelectRow={openReport}
-            onCheckboxPress={toggleTransaction}
-            onAllCheckboxPress={toggleAllTransactions}
-            canSelectMultiple={type !== CONST.SEARCH.DATA_TYPES.CHAT && canSelectMultiple}
-            shouldPreventLongPressRow={isChat}
-            SearchTableHeader={
-                !isLargeScreenWidth ? undefined : (
-                    <SearchTableHeader
-                        data={searchResults?.data}
-                        metadata={searchResults?.search}
-                        onSortPress={onSortPress}
-                        sortOrder={sortOrder}
-                        sortBy={sortBy}
-                        shouldShowYear={shouldShowYear}
-                        shouldShowSorting={shouldShowSorting}
-                    />
-                )
-            }
-            contentContainerStyle={[contentContainerStyle, styles.pb3]}
-            containerStyle={[styles.pv0, type === CONST.SEARCH.DATA_TYPES.CHAT && !isSmallScreenWidth && styles.pt3]}
-            shouldPreventDefaultFocusOnSelectRow={!canUseTouchScreen()}
-            onScroll={onSearchListScroll}
-            onEndReachedThreshold={0.75}
-            onEndReached={fetchMoreResults}
-            ListFooterComponent={
-                shouldShowLoadingMoreItems ? (
-                    <SearchRowSkeleton
-                        shouldAnimate
-                        fixedNumItems={5}
-                    />
-                ) : undefined
-            }
-        />
+        <SearchScopeProvider isOnSearch>
```

Comment:
why are wrapping the SearchList into this scope provider and then not use the scope provider anywhere else? It feels like a reusable code that does something weird with searchContext and then the only use case is inside search - that already has access to Search context.